Swift Charts Examples/Charts/RangeCharts/Heart Rate/HeartRateRangeChart.swift

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright © 2022 Swift Charts Examples.
+// Open Source - MIT License
+
+import SwiftUI
+import Charts
+
+struct HeartRateRangeChartDetail: View {
+    @State var barWidth = 10.0
+    @State var chartColor: Color = .red
+    @State var isShowingPoints: Bool = false
+    
+    var body: some View {
+        List {
+            Section(header: makeHeader()) {
+                Chart(HeartRateData.lastWeek, id: \.weekday) {
+                    BarMark(
+                        x: .value("Day", $0.weekday, unit: .day),
+                        yStart: .value("BPM Min", $0.dailyMin),
+                        yEnd: .value("BPM Max", $0.dailyMax),
+                        width: .fixed(barWidth)
+                    )
+                    .clipShape(Capsule())
+                    .foregroundStyle(chartColor.gradient)
+                }
+                .chartXAxis {
+                    AxisMarks(values: .stride(by: ChartStrideBy.day.time)) { _ in
+                        AxisTick()
+                        AxisGridLine()
+                        AxisValueLabel(format: .dateTime.weekday(.abbreviated), centered: true)
```

Comment:
The value in Apple's version is aligned to the tick/grid line
```suggestion
                        AxisValueLabel(format: .dateTime.weekday(.abbreviated))
```